### PR TITLE
[docs-only] Update testing docs

### LINKED ISSUE #6263: add documentation to run e2e tests
### Description
Currently web docs https://owncloud.dev/clients/web/ only contains documentation for:
- running acceptances tests
- running acceptance tests manually

It will be very helpful if we also have the docs for running the e2e tests

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Related Issue
- Fixes #6263

### CODE REVIEW

File: docs/testing/testing.md [hunk 1]
```diff
@@ -32,10 +32,92 @@ $ yarn test:integration
...
+- `RETRY=n` to retry failures `n` times
+- `SLOW_MO=n` to slow the execution time by `n` milliseconds
+- `TIMEOUT=n` to set tests to timeout after `n` milliseconds
+- `HEADLESS=bool` to open the browser while the tests run (defaults to true => headless mode)
+- `BROWSER=name` to run tests against a specific browser. Defaults to Chrome, available are Chrome, Firefox, Webkit, Chromium
+
+For debugging reasons, you may want to record a video or traces of your test run. 
+Again, you can use the following ENV variables in your command:
+
+- `REPORT_DIR=another/path` to set a directory for your recorded files
```
Comment (fschade) [inline docs/testing/testing.md:106]: can we mention that by default the path is `reports`

File: docs/testing/testing.md [hunk 2]
```diff
@@ -32,10 +32,92 @@ $ yarn test:integration
...
+- `REPORT_VIDEO=true` to record a video of the test run
+- `REPORT_HAR=true` to save request information from the test run
+- `REPORT_TRACING=true` to record traces from the test run
+
+To then open e.g. the tracing, run
+
+```shell
+$ npx playwright show-trace path/to/file.zip
```
Comment (fschade) [inline docs/testing/testing.md:114]: maybe we have to mention somehow that the files live inside the reports dir... e.g. `npx playwright show-trace $REPORT_DIR/to/file.zip`

**Review by fschade — APPROVED** [review]
just a few little things, expect that 🎸

File: docs/testing/testing.md (hunk 1 above)
Comment (pascalwengerter) [reply docs/testing/testing.md:106]: Will follow up on the feedback, was too quick to merge 😅 😉

Comment (fschade) [reply docs/testing/testing.md:106]: as said, totally optional and not a show stopper in any meaning, thanks for taking care

**Pull request merged**